extensions/amp-lightbox/1.0/base-element.js

```diff
@@ -38,6 +38,11 @@ export class BaseElement extends PreactBaseElement {
     });
   }
 
+  /** @override */
+  updatePropsForRendering(props) {
+    props['closeButtonAs'] = () => props['closeButton'];
+  }
```

Comment:
FMI: What is the purpose of this method?

Reply:
Discussed in [this thread](https://github.com/ampproject/amphtml/pull/33105#discussion_r588601246)

Reply:
Got it, will follow up in the linked thread.